src/App.tsx

```diff
@@ -1,25 +1,40 @@
-import React from 'react';
+import React, { ChangeEvent, useState } from 'react';
 import './App.css';
 import { getNumbers } from './utils';
 
-// eslint-disable-next-line @typescript-eslint/no-unused-vars
-const items = getNumbers(1, 42).map(n => `Item ${n}`);
+import { Pagination } from './components/Pagination';
 
 export const App: React.FC = () => {
+  const totalAmount = 42;
+  const items = getNumbers(1, 42).map(n => `Item ${n}`);
+  const [value, setValue] = useState(5);
+  const [sortPage, setSortPage] = useState(1);
+
+  const handlePageChange = (page: number) => {
+    setSortPage(page);
+  };
+
   return (
     <div className="container">
       <h1>Items with Pagination</h1>
 
       <p className="lead" data-cy="info">
-        Page 1 (items 1 - 5 of 42)
+        Page {sortPage} (items {(sortPage - 1) * value + 1} -{' '}
+        {Math.min(sortPage * value, totalAmount)} of {totalAmount})
```

Comment:
consider moving `{(sortPage - 1) * value + 1` and  `Math.min(sortPage * value, totalAmount)` to separate consts and reuse here